modules/osd-monitoring-querying-metrics-for-all-projects-as-an-administrator.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * monitoring/osd-managing-metrics.adoc
+
+[id="querying-metrics-for-all-projects-as-an-administrator_{context}"]
+= Querying metrics for all projects as an administrator
+
+As a `dedicated-admin` or as a user with view permissions for all projects, you can access metrics for all default {product-title} and user-defined projects in the Metrics UI.
+
+[NOTE]
+====
+Only dedicated administrators have access to the third-party UIs provided with {product-title} Monitoring.
+====
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `dedicated-admin` role or with view permissions for all projects.
+
+.Procedure
+
+. From the *Developer* perspective in the OpenShift web console, select *Monitoring* -> *Metrics*.
```

Comment:
I suppose here is *Administrator* perspective, if *Developer* perspective, no *Metrics* under *Monitoring*

Reply:
Changed it.